config/_default/menus/main.en.yaml

```diff
@@ -4588,6 +4588,11 @@ menu:
       parent: cloud_siem
       identifier: cloud_siem_investigator
       weight: 6
+    - name: Historical Jobs
+      url: security/cloud_siem/historical_jobs
+      parent: cloud_siem
+      identifier: cloud_siem_log_historical_jobs
+      weight: 7
```

Comment:
You might have to change the Guides weight to 8. In staging, Guides is above Historical Jobs